system/include/libc/bits

```diff
@@ -1 +0,0 @@
-../arch/emscripten/bits
```

Comment:
This must be some fuckup with line ending conversion by git on my client I think. I never touched the file. I think this happend when I synced my repository to the current version of kripken/incoming on my windows client.